EdgeReachTech/adgereachtech-web-bn
# Userbranch

What does this PR do?
This PR adds a new feature for user authentication using JWT tokens.

Description of Task to be completed?
Added a new JWT verification.
Refactored the login and registration endpoints to generate and return JWT tokens.

How should this be manually tested?

- Clone the repository and check out this branch.
- Run **npm install** to install dependencies.
- Run **npm run dev** to start the application.
- Use Postman to test the login endpoint with valid credentials.
- Verify that a JWT token is returned in the response.

Any background context you want to provide?
This feature is part of the ongoing effort to improve the security of our application by implementing modern authentication methods.

Questions:
Include any questions you might have for the reviewers. This can be related to the implementation, potential improvements, or any uncertainties you might have.

Example:

> Are there any edge cases I might have missed in the token generation process?
> Should we consider adding more detailed error handling for token verification failures?
